chevah/empirical/tests/normal/test_mockup.py

```diff
@@ -316,8 +316,10 @@ def test_bytes_string_conversion(self):
         with self.assertRaises(UnicodeDecodeError):
             value.decode(encoding='utf-8')
 
-        with self.assertRaises(UnicodeDecodeError):
-            value.decode(encoding='utf-16')
+        # UTF-16 never fails on AIX.
```

Comment:
this is a pretty bad comment... never fails... but why?
It should have a FIXME reference with a folloup up actions.

But I feel like it is a bug in this new code

```
$ ssh chevah@172.20.4.36
$ cd sftpplus-aix53-ppc-3.3.0
$ ./bin/python 
Python 2.7.8 (default, Jul 31 2015, 13:08:36) [C] on aix5
Type "help", "copyright", "credits" or "license" for more information.
>>> 'a'.decode('utf-16')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/chevah/sftpplus-aix53-ppc-3.3.0/lib/python2.7/encodings/utf_16.py", line 16, in decode
    return codecs.utf_16_decode(input, errors, True)
UnicodeDecodeError: 'utf16' codec can't decode byte 0x61 in position 0: truncated data
```